pkg/kopia/command/blob.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+type BlobListCommandCommandArgs struct {
+	*CommandArgs
+}
+
+// BlobList returns the kopia command for listing blobs in the repository with their sizes
+func BlobList(blobListArgs BlobListCommandCommandArgs) []string {
```

Comment:
Overall everything looks good but I feel the name `blobListArgs` is sort of redundant. We have enough information to infer that these args are blob list command args 😄 So, I would just simplify things by renaming this to `args`. Same comment for almost all the other command builders 👍🏼